### ISSUE TITLE
Help "files" shouldn't be in Recent

### ISSUE THREAD

**Issue by vr8hub** [typora/typora-issues#1371]
Typora .9.9.13.6 on OSX 10.12.6.

When viewing various Help elements (Markdown reference, Custom Themes, etc.), it appears that the help is contained in files. That's fine, but when viewing the help, Typora is then inserting those "files" into the "Recent" list.

Recent should only contain *user* files. Whatever means Typora uses to display Help should never appear in Recent. We don't open that "file" — we just viewed some Help.

**Comment by abnerlee** [comment]
Help info in Typora are markdown files since it is a quick way to show what Typora is capable of to user. So they are "opening" those files, and recording them is acceptable in my mind.

Similar behavior is Chrome, you could view help from `Help` -> `Help Center`, which opens a webpage in Chrome, and thet page is recorded in Chrome's history.

**Comment by vr8hub** [comment]
It's not acceptable in my mind. :)

I don't use Chrome. And even if I did, browsers keep all history, so the presence of a Help page isn't taking up space that could be used by something else. Typora's is. If I look at three help pages, that's 30% less "recent's" that I have. That's damaging to my history.

We shouldn't be penalized just because you decided to implement Help as files.

**Comment by AlexPasternak** [comment]
However, as a counterpoint, just noticed that Notepad++ on Windows, which is very popular and respected, does puts its own "What's New" log, that it shows automatically on version updates, in its MRU list.

<img width="318" alt="untitled 3" src="https://user-images.githubusercontent.com/1819485/38256840-2ea6416e-372d-11e8-92d3-9d90648f6512.png">

**Comment by abnerlee** [comment]
Since reading help content is just *open* related md file, it should be all feature enabled.

User may also use Typora to open some readme files which won't be used anymore, but they should still be recorded in history, Typora's *readme* file should be nothing special.

**Comment by vr8hub** [comment]
It should be pretty obvious that's not the same at all. If I open a file, then of course it should be on history, regardless of what type of file it is. That's what history is for — to show files I opened.

But I didn't open a file. I opened Help (not a readme; I'm not sure how this discussion morphed into readme's). You opened a file to show me the help. If you had to open five files to show me the Help, that would be half the recent list.

Again, it's pretty simple. The recent list is for files I open. I didn't open the Help file, Typora did. They shouldn't be on the list.
